hexa/pipelines/schema/mutations.py

```diff
@@ -43,11 +44,24 @@ def resolve_create_pipeline(_, info, **kwargs):
         }
 
     try:
-        pipeline = Pipeline.objects.create(
-            code=input["code"],
-            name=input.get("name"),
-            workspace=workspace,
-        )
+        data = {
+            "code": input["code"],
+            "name": input.get("name"),
+            "workspace": workspace,
+        }
+        if input.get("notebookPath", None) is not None:
+            data["type"] = PipelineType.NOTEBOOK
+            data["notebook_path"] = input["notebookPath"]
+            # we need to check if the notebook path exist in the workspace bucket
+            get_bucket_object(workspace.bucket_name, data["notebook_path"])
+        else:
+            data["type"] = PipelineType.ZIPFILE
+
+        pipeline = Pipeline.objects.create(**data)
```

Comment:
I think we need to associate the pipeline(type=Notebook) with a PipelineVersion @qgerome